### PR TITLE
Updated the docs 

### PR DESCRIPTION
**Pull request by nikeedev**

### CODE REVIEW

**Review by nikeedev — COMMENTED** [review]
update a bit

File: addon/addon.json [hunk 1]
```diff
@@ -1,7 +1,5 @@
 [
```
Comment (StioStudio) [inline addon/addon.json]: Are you deleting them?

**Comment by StioStudio** [comment]
Is this general updates? (Like no new addon but many different changes)

File: addon/addon.json (hunk 1 above)
Comment (nikeedev) [reply addon/addon.json]: Yeah, i did it on my side, as I didn’t have you updates before you committed a big update with that fix

**Comment by nikeedev** [comment]
> Is this general updates? (Like no new addon but many different changes)

yep, i changed mostly the docs, so there isn't a lot different but...its updated 👍

**Review by StioStudio — APPROVED** [review]
That should be good

**Review by nikeedev — COMMENTED** [review]
Okay

**Review by StioStudio — APPROVED** [review]

**Pull request merged**